.github/workflows/framework-kind.yml

```
@@ -0,0 +1,14 @@
+name: Framework
+
+on:
+  push:
+    tags:
+      - v0.**
```

Good idea to run the tests on tag pushes! Is there a reason for this RegEx or could we make it more generic?

```suggestion
      - v[0-9]+.[0-9]+.[0-9]+
```

---

I didn't have anything particular in mind so I just used something that works. I'm open to making it more generic!